microshift_configuring/microshift-workload-partitioning.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-workload-partitioning"]
+= Workload partitioning
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-workload-partitioning
+
+toc::[]
+
+Workload partitioning separates compute node CPU resources into distinct CPU sets. The primary objective is to limit the amount of CPU usage for all control plane components which reserves rest of the device CPU resources for workloads of the user.
```

Comment:
`compute node CPU`, in Microshift we don't have multiple nodes, we have one node which also acts as compute , the rest of the  the goal is correct .

Reply:
done